apps/docs/pages/guides/realtime/bring-your-own-database.mdx

```diff
@@ -31,6 +31,8 @@ Realtime relies on Postgres' logical replication functionality to get database c
 - `max_replication_slots`: we recommend `10` because Realtime requires a few slots plus the slots you'll need for your non-Realtime logical replication needs.
```

Comment:
Should we change this recommendation of `10` based on the new info?

Reply:
It's fine to leave at 10 for external databases I think (though really it should delve into `existing replication slots + margin for realtime + margin for your growth` or similar)